src/AnimalList/AnimalList.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './AnimalList.css';
+import backgroundImg from '../background.png';
+
+export default function AnimalList({ animals }) {
+  return (
+    <div className='back' style={{ backgroundImage: `url(${backgroundImg})` }}>
+      {
+        animals.map((animal, i) => <div style={{
```

Comment:
This is the perfect spot to call your Animals component to render out the animal info - instead of hard coding this div.